src/prog/main.F90

```diff
@@ -1778,6 +1778,18 @@ subroutine parseArguments(env, args, inputFile, paramFile, lgrad, &
          case ('--alpha')
             call set_elprop('alpha')
 
+         case ('--raman')
+            call set_elprop('raman')
```

Comment:
Similar to the comment above, a switch to turn on PTB should be added:
```
            call set_exttyp('ptb')
            if (.not. get_xtb_feature('tblite')) then
               call ptb_feature_not_implemented(env)
               return
            endif
```

Reply:
The intended principle behind `--raman` is in my opinion the combination of the flags `--ptb --hess --alpha` together with Raman-specific options like temperature and frequency of incident light.

Reply:
(The automatic redirect to `--ptb` follows from the fact that PTB is currently the only implemented method capable of calculating polarizabilities.)

Reply:
Good idea, but I'd still preserve `--ptb` as a requirement in case of the Raman implementation with any other tight-binding method.

Reply:
Also fine for me. However, it should be checked whether an error is produced if a call like `--raman --gfn 2` or similar is used. (Something like: "Raman not implemented with this method".)